src/app/components/EditorInterface.tsx

```diff
@@ -304,16 +305,21 @@ const EditorInterface = ({ settings, token }: IEditorInterface) => {
               background: `${appState?.appState.editorColor}`,
             }}
           >
-            <When condition={!!appState?.appState.bannerImgUrl}>
-              <img
-                className='w-full object-fill xl:object-cover'
+            {!!appState?.appState.bannerImgUrl && bannerImage ? (
+              <Image
+                className='w-full'
                 src={bannerImage}
                 alt='banner image'
+                width={0}
```

Comment:
yes, because width and height are the required props for next/image. This is a hack to set the image's width and height according to the design. For your reference please check [THIS](https://stackoverflow.com/a/76008677/11269810) StackOverflow's answer. @sazanrjb